### PR TITLE
F#src5190 removesleeps

### PR DESCRIPTION
**Pull request by giusebar**

### CODE REVIEW

File: ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/launch-everything-bimanual.j2 [hunk 1]
```diff
@@ -1,10 +1,35 @@
 #jinja2: trim_blocks:False
```
Comment (mikramarc) [inline ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/launch-everything-bimanual.j2:1]: same stuff in this file

Comment (mikramarc) [reply ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/launch-everything-bimanual.j2:1]: and all files below

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: ansible/roles/products/hand-e/server/desktop-icons/tasks/main.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@
   when: "'extra_lite' in product"
 
 - name: Install desktop icon for launching everything for Shadow Right Hand
-  import_tasks: default-icon-no-terminator.yml
+  import_tasks: default-icon.yml
```
Comment (ToivoS) [inline ansible/roles/products/hand-e/server/desktop-icons/tasks/main.yml:28]: Why this change: default-icon-no-terminator.yml -> default-icon.yml

**Review by ToivoS — CHANGES_REQUESTED** [review]

**Review by ToivoS — CHANGES_REQUESTED** [review]
Why this change: default-icon-no-terminator.yml -> default-icon.yml

**Review by mikramarc — APPROVED** [review]

**Review by ToivoS — APPROVED** [review]

**Pull request merged**